PinpointKit/PinpointKit/Sources/BoxAnnotationView.swift

```diff
@@ -16,7 +16,12 @@ public class BoxAnnotationView: AnnotationView {
     /// The corresponding annotation.
     var annotation: BoxAnnotation? {
         didSet {
-            layer.shadowPath = annotation.flatMap(PathForDrawingBoxAnnotation)?.cgPath
+            if let annotation = annotation {
+                layer.shadowPath = self.dynamicType.path(forDrawing: annotation)?.cgPath
```

Comment:
Ah, yeah, it did read as a noun for me. `path(toDraw: annotation)`, `path(for: annotation)`, or `path(of: annotation)`?